firefox-ios/Client/Telemetry/HomepageTelemetry.swift

```diff
@@ -5,9 +5,25 @@
 import Foundation
 import Glean
 
+protocol GleanWrapperProtocol {
```

Comment:
The logical steps to implement the ticket are good here, but we should use the `GleanWrapper` protocol and the class `DefaultGleanWrapper` that already exists in the code base instead of creating a new Glean wrapper protocol and class. 

To use the existing `GleanWrapper`, since the `PrivateModeToggleExtra` is of type `EventMetricType`, it means the `HomepageTelemetry` will call the `recordEvent<ExtraObject>(for metric:extras:)` function to pass in the extra event in there.

Reply:
okay, will give you an PR soon.